scripts/validate_settings.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import argparse
+import warnings
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    from camayoc.config import get_settings
```

Comment:
why ignore those warnings? 🤔

Reply:
This is specifically for [a warning issued by `get_settings_files`](https://github.com/quipucords/camayoc/blob/master/camayoc/config.py#L47-L53). That function will be run on import time, and warning tells you there's no camayoc config file in XDG directories.

I imagine following use-case for this script: you clone camayoc repository, copy example config and try to adjust it to your environment. You run a script to tell you if your config will work or not. Only when it passes, you copy it to `~/.config/camayoc/` to use it. So there's no point in script telling you there's no config file in `~/.config/` - you know that, you are working on this right now.